website/content/api-docs/query.mdx

```diff
@@ -194,14 +194,25 @@ The table below shows this endpoint's support for
       determine the priority.
 
     - `Datacenters` `(array<string>: nil)` - Specifies a fixed list of remote
-      datacenters to forward the query to if there are no healthy nodes in the
+      datacenters to forward the query to when there are no healthy nodes in the
```

Comment:
I recommend clarifying that the `Datacenters` field can only be used to specify WAN federated datacenters as failover targets. And that to specify a cluster peered datacenter or admin partition as a failover target, they need to use the `Targets` field instead.